owncloudApp/src/main/java/com/owncloud/android/presentation/sharing/ShareActivity.kt

```diff
@@ -154,7 +154,10 @@ class ShareActivity : FileActivity(), ShareFragmentListener {
                     is UIResult.Loading -> {
                         showLoadingDialog(R.string.common_loading)
                     }
-                    is UIResult.Success -> {}
+                    is UIResult.Success -> {
+                        // needs to trigger refresh after creation because creation request returns wrong path [issue: https://github.com/owncloud/ocis/issues/9746] resulting list not getting updated.
```

Comment:
Please could you remove the link in the comment. We will link in the pull request. Also the first word of the comment should be in capital letter. Fix this and ready to go 🥇

Reply:
And please, fix indentation.

Reply:
@jesmrec I'm using reformat code (Ctrl+Alt+L) , is there any other I should be formatting this?

Reply:
that's right @parneet-guraya , not i see it correct 👍